lib/helpers/html.js

```diff
@@ -76,3 +76,13 @@ exports.ol = function(context, options) {
     return "<li>" + (options.fn(item)) + "</li>";
   }).join("\n") + "</ol>";
 };
+
+
+/**
+ * Remove all HTML tags from the nested context
+ * @param  {[type]} options [description]
+ * @return {[type]}
+ */
+exports.striptags = function(options) {
+  return options.fn(this).replace(/<[^>]*?>/g, "");
```

Comment:
I know it wasn't there when you did this PR, but maybe you can update this to use the `utils.sanitize` method:

```js
return utils.sanitize(options.fn(this));
```